scraper/src/fcc2zim/build.py

```diff
@@ -0,0 +1,138 @@
+import json
+import pathlib
+from collections import OrderedDict
+from datetime import datetime
+
+from zimscraperlib.zim import Creator
+
+from fcc2zim.constants import FCC_LANG_MAP, VERSION, Global
+
+logo_path = pathlib.Path(__file__).parent.joinpath("assets", "fcc_48.png")
+
+
+def build_curriculum_redirects(curriculum_dist_dir: pathlib.Path, language: str):
```

Comment:
dosctring

Reply:
Unfortunately this is not from me, and I cannot explain much about it.

Reply:
Sorry, I should have added some docs to this method. The Vite app uses it's own router to navigate. So you'll end up with a single HTML file, but we need an URL for each item in the curriculum for the zim search to work. This builds a simple redirect of the URL to the Vite hash URL. Alternatively we would need to involve the Vite build/config process to create a static page for each piece of content, but this would mean we couldn't just build the Vite client and then add arbitrary content, but would need to build the Vite client for each curriculum set. I was hoping to just reuse the Vite front-end client build for any curriculum sets, but it's possible to do it another way.

Reply:
This is fine for me